### ISSUE TITLE
A11y_.NET Core_Recipes App My Recipes New_New Recipe_Voiceover: Voice over is not announcing the "Review" label for slider, Minus, Plus buttons.

### ISSUE THREAD

**Issue by Swati1700** [dotnet/maui#15595]
### Description

[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NET Core[MAUI]-MAC-June2023; #.NET Core; #WCAG1.3.1; #Element:Label; #ScreenReader; #Voiceover; #MAC; 

### Environment Details:​
Application Name: Rachel's Recipe
macOS: Ventura 13.3
Screen Reader: Voiceover

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.TAB till "My Recipes" and select.
3.TAB till "New" button and select.
4."New Recipe" Screen will open.
5. Start Voiceover.
6. TAB till Slider and increment, decrement buttons.
7. Observe voiceover is not announcing "Review" label along with slider information.

### Actual Results:

In "New Recipe" screen voice over is not announcing the "Review" label for slider, Minus, Plus buttons. 

### Expected Results:

In "New Recipe" screen voice over should announce the "Review" label for slider, Minus, Plus buttons. 

### User Impact:
Voiceover users will face difficulties while navigating to the slider and increment, decrement buttons and voiceover is not announcing the label for it.

**Note:** The issue is discussed with the PWD users and as per discussion the label should be announce by the voiceover for slider so the user will understand the purpose of the slider and "+", "-" button. As discussed with the PWD user this is logged under Sev2.

### Attachment:
![issu22](https://github.com/dotnet/maui/assets/86710888/1e6310a1-bcab-49f2-bfde-b6070f3bb7ea)

### Link to public reproduction project repository

Test

### Version with bug

7.0.49

### Last version that worked well

6.0

### Affected platforms

macOS

### Affected platform versions

Ventura 13.3

**Comment by rachelkang** [comment]
Updated slider semantics. More work needed to add semantics to individual stepper buttons

**Comment by Swati1700** [comment]
Verified the issue in the application Rachel's Recipes version 2(2). Issue is fixes and does not repro, hence closing this bug.
Please find below closing attachment.

https://github.com/dotnet/maui/assets/86710888/08c20783-5ce5-466e-8305-bc56d1957a35
